### PR TITLE
[Proteomics] Import Introductory slide deck from @foellmelanie

### PR DESCRIPTION
**Pull request by shiltemann**
Hi @foellmelanie! I made a first pass at adding your slides to the GTN format, please feel free to change anything you want.

I left one TODO in there for you, that schematic did not display nicely for me because I don't have proper Powerpoint here, so would be great if you can take a screenshot of it and add it in

Let me know what you think and if anything should change! 

Oh, and go ahead and add speaker notes if you would like to use the slides-to-video feature :)

### CODE REVIEW

File: topics/proteomics/slides/introduction.html [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+]
+
+.right-column40[
+**Identification options:**
+
+- Manual interpretation
+
+- De novo sequencing
+
+- Matching to in silico spectra (generated from fasta database)
```
Comment (subinamehta) [inline topics/proteomics/slides/introduction.html:142]: - Matching to an in-silico spectra (generated from fasta database)

File: topics/proteomics/slides/introduction.html [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+#  MSstats Software
+
+.left-column50[
+
+.image-100[
+  ![]({{site.baseurl}}/topics/proteomics/images/intro-slides/msstats-paper.png)
+]
+
+- Two Bioconductor R Packages
+  - Msstats & MSstatsTMT
```
Comment (subinamehta) [inline topics/proteomics/slides/introduction.html:340]: - MSstats & MSstatsTMT

**Review by subinamehta — COMMENTED** [review]
I have added few minor corrections.

**Comment by shiltemann** [comment]
Thanks for the review @subinamehta! :+1:

**Review by hexylena — APPROVED** [review]

**Pull request merged**